### DISCUSSION TITLE
When using multiple configuration, what's the recommended way to save the account "type"?

### DISCUSSION THREAD

**Discussion by nicolas-besnard** [jeremyevans/rodauth#233] [General]
I'm currently using `rodauth-rails` in my project, and I would like to have different `accounts` table (`accounts` and `admin_accounts`) to have a clear separation and avoid problem in the feature.

In `rodauth-rails` documentation, the [Wiki][wiki] propose to have one `accounts` table and use a different one for "account types).

You can also have a new column in your `accounts` table to old this info.

If I were to use Devise, I could easily configure 2 different models. With `rodauth`, it requires to redefine a lot configuration (like `verify_account_table`, `accounts_table` etc...)

What would be your recommendation/proposal for a safe multi-type accounts using multiple configuration ?

[wiki]: https://github.com/janko/rodauth-rails/wiki/Account-Types

**Comment by jeremyevans** [comment]
This should be as simple as:

```ruby
accounts_table do
  if admin_criteria
    :admin_accounts
  else
    :accounts
  end
end
```
